flaml/onlineml/trial.py

```diff
@@ -369,53 +397,59 @@ def train_eval_model_online(self, data_sample, y_pred):
+        )
         # udpate sample size, sum of loss, and cost
         data_sample_size = 1
         bound_of_range = self._y_max_observed - self._y_min_observed
         if bound_of_range == 0:
             bound_of_range = 1.0
-        self.result.update_result(new_loss,
-                                  VowpalWabbitTrial.cost_unit * data_sample_size,
-                                  self._dim, bound_of_range)
+        self.result.update_result(
+            new_loss,
+            VowpalWabbitTrial.cost_unit * data_sample_size,
+            self._dim,
+            bound_of_range,
+        )
 
     def predict(self, x):
-        """Predict using the model
-        """
+        """Predict using the model."""
         if self.model is None:
             # initialize self.model and self.result
             self._initialize_vw_model(x)
         return self.model.predict(x)
 
     def _get_loss(self, y_true, y_pred, loss_func_name, y_min_observed, y_max_observed):
         """Get instantaneous loss from y_true and y_pred, and loss_func_name
-            For mae_clip, we clip y_pred in the observed range of y
+        For mae_clip, we clip y_pred in the observed range of y
         """
-        if 'mse' in loss_func_name or 'squared' in loss_func_name:
+        if "mse" in loss_func_name or "squared" in loss_func_name:
             loss_func = mean_squared_error
-        elif 'mae' in loss_func_name or 'absolute' in loss_func_name:
+        elif "mae" in loss_func_name or "absolute" in loss_func_name:
             loss_func = mean_absolute_error
-            if y_min_observed is not None and y_max_observed is not None and \
-               'clip' in loss_func_name:
+            if (
+                y_min_observed is not None
+                and y_max_observed is not None
+                and "clip" in loss_func_name
+            ):
                 # clip y_pred in the observed range of y
                 y_pred = min(y_max_observed, max(y_pred, y_min_observed))
         else:
             raise NotImplementedError
         return loss_func([y_true], [y_pred])
 
     def _update_y_range(self, y):
-        """Maintain running observed minimum and maximum target value
-        """
+        """Maintain running observed minimum and maximum target value"""
         if self._y_min_observed is None or y < self._y_min_observed:
             self._y_min_observed = y
         if self._y_max_observed is None or y > self._y_max_observed:
             self._y_max_observed = y
 
     @staticmethod
-    def _get_dim_from_ns(namespace_feature_dim: dict, namespace_interactions: [set, list]):
-        """Get the dimensionality of the corresponding feature of input namespace set
-        """
+    def _get_dim_from_ns(
+        namespace_feature_dim: dict, namespace_interactions: [set, list]
+    ):
+        """Get the dimensionality of the corresponding feature of input namespace set"""
```

Comment:
`"""Get the dimensionality of the corresponding feature of input namespace set."""`